src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
+@import "./variables";
+
 body {
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  width: $column-width;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  gap: $gap;
+
+  @for $month-length from 28 through 31 {
+    &--#{$month-length} :nth-child(#{$month-length}) {
+      & ~ .calendar__day {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    display: flex;
+    flex-direction: column;
```

Comment:
The flex-direction property is not needed here. The default value of flex-direction is row, which is what we want for the calendar cells.